contents/blog/product-360.md

```diff
@@ -0,0 +1,124 @@
+---
+date: 2022-01-18
+title: I used to think product people aren't needed. I was wrong
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog", "Engineering"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/product-people.png
+featuredImageType: full
+---
+
+When PostHog was fresh out of YC last year, there were two things we focused on: (i) build and (ii) talk to customers. Hiring a product person felt like it would slow us down. Engineers on our team would speak to our users (also engineers) - surely they'd know what's best to prioritize? We didn't want our engineers to be "shielded" from our customers.
+
+By [PostHog's series B](/blog/15-million-series-b), our team showed me that I was totally wrong. Here's why that was the case, what we did about it, and the results.
+
+## Why I used to think this way
+
+My previous experience of product management:
+
+* Here's a list of features the enterprise sales team wants
+* How much pipeline does each of these features influence?
+* _Builds as much of the above as possible whilst trying to manage existing customer requests_
+
+Lead to:
+
+* Lots of initial enterprise contracts
+* A product users find frustrating to use
+* No word of mouth growth
+* Customer churn in the long term
+
+## It starts with expectation setting
+
+<a href="https://imgflip.com/i/61pr1n"><img src="https://i.imgflip.com/61pr1n.jpg" title="made at imgflip.com"/></a><div><a href="https://imgflip.com/memegenerator">from Imgflip Meme Generator</a></div><br />
```

Comment:
I think this meme has been mis-used - the point of it is that Morpheus is sharing a mindblowing insight with you. Reading this, it sounds like you are saying 'The insight we found is that the board your product manager'.  

Something like this I think makes the point better:
<img width="474" alt="Screenshot 2022-02-03 at 12 20 04" src="https://user-images.githubusercontent.com/70321811/152341659-77acbc63-5b9c-4c7f-8233-971216e25c83.png">

Reply:
@jamesefhawkins happy to use this one? I'll upload this as I'd prefer not to load stuff directly from imgflip

Reply:
I've switching this in now